monitoring/reviewing-monitoring-dashboards.adoc

```diff
@@ -20,15 +20,15 @@ In the *Administrator* perspective you can access dashboards for core {product-t
 .Example dashboard in the Administrator perspective
 image::monitoring-dashboard-administrator.png[]
 
-In the *Developer* perspective you can access dashboards that provide the following statistics for a selected project:
+In the *Developer* perspective, you can access Kubernetes compute resources dashboards that provide the following statistics for a selected project:
```

Comment:
```suggestion
Use the *Developer* perspective to access Kubernetes compute resources dashboards that provide the following application metrics for a selected project:
```

Trying to be consistent with the usage of words.

Reply:
Suggestion for L11: 
Use the *Administrator* perspective to access dashboards for the core {product-title} components, including the following <items/components>:

Suggest using a complete sentence for the lead-in sentence to reduce ambiguity and support translation. Refer to: https://redhat-documentation.github.io/supplementary-style-guide/#lead-in-sentences

Reply:
This seems to be a better lead-in sentence, "Use the *Administrator* perspective to access dashboards for the core {product-title} components, including the following items:"

Cc @vikram-raj